core/common/assets/js/api/index.js

```diff
@@ -51,6 +52,10 @@ export default class API {
 			hookUI,
 		};
 
+		this.extras = {
```

Comment:
Suggestion: Maybe `this.hashCommandsManager`?

Reply:
Since hash is a browser thing/external staff its abnormal... Imagine we use it in backend ( nodejs )...

And its way too different from other properties in this level.